src/e3/os/platform.py

```diff
@@ -167,15 +170,20 @@ def os_version(cls) -> tuple[str, str]:
             if "redhat" in name or "red hat" in name:  # os-specific
```

Comment:
Rather than repeating `ull_version_number = cls.ld_info["version"]`, what about setting it once and for all (before or after the `if` statement) like:
```python
distro_version: str = cls.ld_info["version"] if "version" in cls.ld_info else ""
```
I feel like the `version_number` could be set this way too.
I do not like the `full_version_number` name because the type of the variable seems to be a string.